DataGrid/Gantt: Rewrite PDF troubleshooting topic (#7630)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/15 PDF Export Issues/00 Export Unicode Characters/00 Gantt/00 Gantt.md

When exporting Gantt data to PDF, some characters may be displayed incorrectly in the generated file. 

> PDF supports a limited set of fonts and they are all limited to the ASCII-codepage. If you want to use UTF-8 characters, you need to supply a font that includes necessary glyphs. This tutorial demonstrates how to add custom fonts to your Gantt PDF exports.

Reviewer: See my comment above regarding "ASCII-codepage".

Author: Same here, we can use "characters" instead of "codepage" to simplify a touch.
```suggestion
PDF supports a limited set of fonts and they are all limited to the ASCII characters. If you want to use UTF-8 characters, you need to supply a font that includes necessary glyphs. This tutorial demonstrates how to add custom fonts to your Gantt PDF exports.
```

Reviewer: I'd like to modify your suggestion here as well:

```suggestion
PDF supports a set of fonts that are limited to the ASCII characters. If you want to use UTF-8 characters, you need to supply a font that includes the required glyphs. This tutorial demonstrates how to add custom fonts to your Gantt PDF exports.
```
